packages/rke2-ingress-nginx/generated-changes/patch/templates/admission-webhooks/job-patch/job-patchWebhook.yaml.patch

```diff
@@ -5,7 +5,7 @@
        containers:
          - name: patch
 -          {{- with .Values.controller.admissionWebhooks.patch.image }}
--          image: "{{- if .repository -}}{{ .repository }}{{ else }}{{ .registry }}/{{ .image }}{{- end -}}:{{ .tag }}{{- if (.digest) -}} @{{.digest}} {{- end -}}"
```

Comment:
Several if statements went from `{{- if - }}` to `{{ if }}`, which messed up a bunch of patches